gradle.properties

```diff
@@ -10,6 +10,7 @@ android.useAndroidX=true
 kapt.verbose=true
 kotlin.code.style=official
 kotlin.daemon.jvmargs=-Xmx8g
+kotlin.incremental.useClasspathSnapshot=true
```

Comment:
https://blog.jetbrains.com/kotlin/2022/07/a-new-approach-to-incremental-compilation-in-kotlin/